tests/e2e/tests/critical-path/tree-view/tree-view.e2e.ts

```diff
@@ -32,56 +35,57 @@ test
         await t.click(browserPage.treeViewButton);
         await t.eval(() => location.reload());
         //Verify that "Tree view" mode enabled state is saved
-        await t.expect(browserPage.treeViewArea.visible).notOk('The tree view is not displayed');
+        await t.expect(browserPage.treeViewArea.visible).ok('The tree view is displayed');
     });
 test
     .meta({ rte: rte.standalone })
     ('Verify that user can see DB is automatically scanned by 10K keys in the background, user can see the number of keys scanned and use the "Scan More" button to search per another 10000 keys', async t => {
         await t.click(browserPage.treeViewButton);
         //Verify the scanned value
-        await t.expect(browserPage.treeViewScannedValue.textContent).contains('Scanned 10 000', 'The database is automatically scanned by 10K keys');
+        await t.expect(browserPage.scannedValue.textContent).eql('10 000', 'The database is automatically scanned by 10K keys');
         //Verify that user can use the "Scan More" button to search per another 10000 keys
         await t.click(browserPage.scanMoreButton);
-        await t.expect(browserPage.treeViewScannedValue.textContent).contains('Scanned 20 000', 'The database is automatically scanned by 10K keys');
+        await t.expect(browserPage.scannedValue.textContent).contains('20 000', 'The database is automatically scanned by 10K keys');
     });
 test
     .meta({ rte: rte.standalone })
-    ('Verify that user can see that “:” (colon) used as a default separator for namespaces', async t => {
+    ('Verify that user can see that “:” (colon) used as a default separator for namespaces and see the number of keys found per each namespace', async t => {
```

Comment:
Maybe it would be better to move this test to regression suit?

Reply:
done